### DISCUSSION TITLE
Is there an example for variogram modelling/fitting?

### DISCUSSION THREAD

**Discussion by flydream0428** [GeoStat-Framework/PyKrige#251] [Q&A] [answered]
Hi, I am trying to find an example to apply PyKrige to some real-world data. Is there such an example around? I may have missed something, but I cannot find it in the doc. For example, I cannot find an example to discuss how to model a suitable variogram. Thanks.

**Comment by MuellerSeb** [comment, marked as answer]
Since PyKrige now is able to use covariance-models from GSTools, you could model your variogram with GSTools and pass this model to the PyKrige class.

Variogram modelling tutorials:
- https://geostat-framework.readthedocs.io/projects/gstools/en/stable/examples/03_variogram/index.html

PyKrige Interface:
- https://geostat-framework.readthedocs.io/projects/gstools/en/stable/examples/05_kriging/02_pykrige_interface.html#sphx-glr-examples-05-kriging-02-pykrige-interface-py
- https://geostat-framework.readthedocs.io/projects/pykrige/en/stable/examples/03_gstools_covmodel.html#sphx-glr-examples-03-gstools-covmodel-py

Does this help?
Cheers, Sebastian

**Comment by flydream0428** [comment]
Thanks, it helps a lot.
